src/reset-password/ResetPasswordPage.jsx

```diff
@@ -20,12 +23,37 @@ const ResetPasswordPage = (props) => {
   const [confirmPasswordInput, setConfirmPasswordValue] = useState('');
   const [passwordValid, setPasswordValidValue] = useState(true);
   const [passwordMatch, setPasswordMatchValue] = useState(true);
+  const [validationMessage, setvalidationMessage] = useState('');
+
+  const validatePassword = async (password) => {
+    const requestConfig = {
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      isPublic: true,
+    };
+    const { data } = await getAuthenticatedHttpClient()
```

Comment:
any reason why we have not written it inside the service.js file?

Reply:
@zainab-amir No specific reason, I believe we can move this into service.

Reply:
@zainab-amir Done, kindly take another look.